xmss.h

```diff
@@ -58,4 +64,11 @@ int xmssmt_sign(unsigned char *sk,
 int xmssmt_sign_open(unsigned char *m, unsigned long long *mlen,
                      const unsigned char *sm, unsigned long long smlen,
                      const unsigned char *pk);
+
+/* 
+ * Returns:
+ * remaining signatures
```

Comment:
nit: "the number of" remaining signatures.

Reply:
I update the description to `The number of remaining signatures`